assembl/static2/css/components/ideas.scss

```diff
@@ -96,3 +96,51 @@
       }
     }
 }
+
+/* iPhone Portrait */
```

Comment:
Well the fix shouldn't just be on iPhone, check on bigger screen (via the chrome simulator) in landscape vs portrait. The changes can still be applied, however.